modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -760,4 +760,25 @@ definition {
 			locator1 = "PRMMDFRequest#STATUS_BUTTON");
 	}
 
+	@summary = "Assert the value for Budget MDF Request"
+	macro viewValueBudget{
+		if (isSet(totalCost)) {
+			AssertTextEquals(
+				key_fieldName = "Total cost",
+				key_fieldValue = ${fieldValue},
+				locator1 = "PRMMDFRequest#VALUE_BUDGET");
+		}
+
+		if (isSet(claimPercent)) {
+			AssertTextEquals(
+				key_fieldName = "Claim Percent",
+				key_fieldValue = ${fieldValue},
+				locator1 = "PRMMDFRequest#VALUE_BUDGET");
+		}
+	}
+
+	@summary = "Verify that the budget value will be cleaned"
+	macro removeBudgetValue {
+		Click(locator1 = "PRMMDFRequest#REMOVE_VALUE_BUDGET_BUTTON");
```

Comment:
Hi @Italo-gabriel, here you can use `Icon#ANY` path, and pass `hr` as value to `key_text`.

```
Click(
	key_text = "hr",
	locator1 = "Icon#ANY");
```

Reply:
Done !